src/utils/uiUtils.ts

```diff
@@ -5,12 +5,25 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import { window, QuickPickItem, QuickPickItemKind, QuickPick } from 'vscode';
+import { window, workspace, QuickPickItem } from 'vscode';
+import { access } from 'fs/promises';
+import {
```

Comment:
These error objects need to be moved into this module as well.

Reply:
Actually, can we move them all into their own module, somewhere separate from `uiUtils`? They're not really UI utilities. Something like `WorkspaceUtils` would probably be more appropriate.